releases/release-1.20/release-notes-draft.md

```diff
@@ -1,178 +1,264 @@
-# Release notes for v1.20.0-beta.2
+# Release notes for v1.20.0-rc.0
 
 [Documentation](https://docs.k8s.io/docs/home)
 # Changelog since v1.19.0
 
+## What's New (Major Themes)
+
+### Dockershim deprecation
```

Comment:
fyi @wilsonehusin – when these eventually make their way over to `k/website`, each of these headers will need a full line underneath them, or the header rendering will break:

```suggestion
### Dockershim deprecation

```

Reply:
😭 okay I'll get on it, thanks Celeste!